packages/shared/wallet/src/settings-side-drawer/settings-side-drawer.tsx

```diff
@@ -200,7 +201,7 @@ const ButtonData = styled(Box)(
     border: ${theme.shape.borderStandard};
     background: ${theme.palette.background.paper};
     font-size: 16px;
```

Comment:
Is font-size supposed to be set here? Is there no corresponding typography object?